src/styles/settings.css

```diff
@@ -21,103 +39,157 @@ input[type='number'] {
   width: 50px;
 }
 
+body {
+  font-size: 75%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
 .container {
   position: relative;
   width: 800px;
   max-width: 800px;
-  background-color: #fff;
-  color: #505050;
+  background-color: var(--main-bg-color);
 }
 
 .tabs {
   display: flex;
   overflow: hidden;
   height: calc(100vh - 35px - 42px);
-  width: 100vw;
+  width: 100%;
   transition: order 300ms ease-in-out;
   align-items: stretch;
   background-color: inherit;
 }
 
 .tab {
   display: none;
-  flex: 1 0 100vw;
+  position: relative;
+  flex: 1 0 100%;
   overflow-y: auto;
   padding: 20px;
   box-sizing: border-box;
 }
 
 .tab.active {
-  display: block;
+  display: flex;
+  flex-direction: column;
 }
 
-.list {
-  padding: 0;
-  margin: 0;
-  list-style: none;
+@media (max-width: 860px) {
+  .tab.active {
+    display: block;
+  }
 }
 
-.list li {
-  position: relative;
-  min-height: 40px;
+.fields {
+  max-width: 800px;
 }
 
-.list li #nag-nanny:checked ~ div {
-  height: auto;
+.field {
+  display: flex;
+  flex-direction: row;
+  padding: 10px 0;
 }
 
-.list li #pomodoro-mode:checked ~ div {
-  height: 220px;
+.field:not(:first-of-type) {
```

Comment:
Nitpick: maybe just improve remove this entry at all? (since the body is commented out) 🤔

Reply:
removed ✅